second_2024_backend/443-invervals/README.md

```
@@ -0,0 +1,56 @@
+1 с
+
+Ограничение памяти
+
+256 МБ
+
+## Пример 1
+
+Ввод | Вывод
+-----|------
+1 1  | 1
+0 5  |
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: no_leading_or_trailing; Actual: trailing_only; Unexpected trailing pipe

(MD055, table-pipe-style)